adapted-articles/J/JesusChristTeachingsof.xml

```diff
@@ -14,77 +14,77 @@
+<p>In <a href="?bref=John.14.16-18">John 14:16-18</a>, Jesus went one step further in identifying himself with the Spirit. He told the disciples that he would give them another Comforter. Then he told them that they should know who this Comforter was because he was, then and there, staying with them and would, in the near future, be in them. Who else but Jesus was staying with them at that time? Then after telling the disciples that the Comforter would come to them, he said, “I am coming to you.” First he said that the Comforter would come to them and be in them, and then in the same breath he said that he would come to them and be in them (see <a href="?bref=John.14.20">14:20</a>). In short, the coming of the Comforter to the disciples was one and the same as the coming of Jesus to the disciples. The Comforter who was dwelling with the disciples that night was the Spirit in Christ; the Comforter who would be in the disciples (after the resurrection) would be Christ in the Spirit.</p>
+<p>On the evening of the resurrection, the Lord Jesus appeared to the disciples and then breathed into them the Holy Spirit. This inbreathing, reminiscent of God’s breathing into Adam the breath of life (<a href="?bref=Gen.2.7">Gn 2:7</a>), became the fulfillment of all that had been promised and anticipated earlier in John’s Gospel. Through this gift (impartation), the disciples were regenerated and indwelt by the Spirit of Jesus Christ. This historical event marked the genesis of the new creation. Jesus could now be realized as the bread of life, the water of life, and the light of life. The believers now possessed his divine, eternal, risen life. From that time forward, Christ as Spirit indwelt his believers. Therefore, in his first letter John could say, “We know he lives in us because the Holy Spirit lives in us” (<a href="?bref=1Jn.3.24">1 Jn 3:24</a>).</p>
```

Comment:
I don't think 'impartation' adds anything here

Reply:
OK. Which alternative below do you prefer, please?
- Through this gift, the disciples…
- Through this act, the disciples…
- Through this, the disciples…

Reply:
1